modules/api/src/main/java/io/fluo/api/config/FluoConfiguration.java

```diff
@@ -667,4 +672,66 @@ public static void setDefaultConfiguration(Configuration config) {
     config.setProperty(MINI_START_ACCUMULO_PROP, MINI_START_ACCUMULO_DEFAULT);
     config.setProperty(MINI_DATA_DIR_PROP, MINI_DATA_DIR_DEFAULT);
   }
+  
+  public FluoConfiguration setPositiveInt(String property, int value) {
```

Comment:
why make these public and part of the API?  I was thinking of these methods as implementation.   If we make them part of API then we have to support them.